src/v/security/scram_algorithm.cc

```diff
@@ -329,17 +329,6 @@ parse_server_final(std::string_view message) {
 
 namespace security {
 
-std::ostream& operator<<(std::ostream& os, const scram_credential& cred) {
```

Comment:
nice. i still can't quite figure out how we ended up with this instance and the one in scram_credential.cc

Reply:
Yeah it's curious, I suppose it could be any of a number of things: concurrent work streams in separate branches that both needed it, snafu in linking causing mayhem in what's defined, etc.